server/schedulers/metrics.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schedulers
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var schedulerCounter = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Namespace: "pd",
+		Subsystem: "scheduler",
+		Name:      "event_count_wip",
```

Comment:
The `_wip` suffix is to avoid conflict with the old one. It will be removed after all schedulers are moved into schedulers directory.